plugins/repository-hdfs/build.gradle

```diff
@@ -269,41 +234,26 @@ if (fixtureSupported) {
   integTestHa.setEnabled(false)
 }
 
-// Secure HDFS testing relies on the Vagrant based Kerberos fixture.
-boolean secureFixtureSupported = false
-if (fixtureSupported) {
-  secureFixtureSupported = project.rootProject.vagrantSupported
-}
-
-if (secureFixtureSupported) {
-  project.check.dependsOn(integTestSecure)
-  project.check.dependsOn(integTestSecureHa)
+project.check.dependsOn(integTestSecure)
+project.check.dependsOn(integTestSecureHa)
```

Comment:
This can simply be `check.dependsOn(integTestSecure, integTestSecureHa)`. I'm not sure there's any benefit in `project.check` vs `check` as well. It's known that everything delegates to `Project` and in both cases there is Groovy dynamic dispatch happening so there's no performance benefit either.